arangod/Cluster/ClusterInfo.h

```diff
@@ -150,19 +154,61 @@ class ClusterInfo final {
     uint64_t hash;
     std::shared_ptr<LogicalCollection> collection;
   };
+
+  struct Hasher {
```

Comment:
Please keep using absl::Hash, it's important for absl containers

Reply:
why is it important?

Reply:
Because absl hashtables performance more dependant of quality of hash functions.

And it already was and it's not difficult to write `absl::HashOf(value)` instead of `std::hash<T>{}(value)`, so why not?